src/sdk/namespace_fs.js

```diff
@@ -890,7 +892,8 @@ class NamespaceFS {
                 const entry_path = path.join(this.bucket_path, r.key);
                 //If entry is outside of bucket, returns stat of symbolic link
                 const use_lstat = !(await this._is_path_in_bucket_boundaries(fs_context, entry_path));
-                r.stat = await nb_native().fs.stat(fs_context, entry_path, { use_lstat });
+                const result_stat = await native_fs_utils.stat_ignore_enoent(fs_context, entry_path, { use_lstat });
+                r.stat = result_stat || r.previous_stat; // must have stat for _get_object_info - we will use the previous stat in case of failure
```

Comment:
if its already a part of r, why do we need to stat it again? if its not a part of r, we might not have stat at all, which will cause an issue with _get_object_info